elasticsearch/templates/statefulset.yaml

```diff
@@ -108,6 +108,20 @@ spec:
       - name: "{{ template "name" . }}"
         image: "{{ .Values.image }}:{{ .Values.imageTag }}"
         imagePullPolicy: "{{ .Values.imagePullPolicy }}"
+        {{- if eq .Values.roles.master "true" }}
+        command:
+          - /bin/sh
+          - -c
+          - |
+            # https://github.com/elastic/helm-charts/issues/63#issuecomment-471096950
+            # Run ES as a background task, and forward SIGTERM to it, then wait for it to exit
+            trap 'kill $(jobs -p)' SIGTERM
+            /usr/local/bin/docker-entrypoint.sh elasticsearch &
```

Comment:
fixed argument passing to es.

Reply:
I found that broken usage of env vars. Maybe I couldn't found this problem because of my custom `elasticsearch.yml` file.

Fixing broken env vars is beyond my knowledge of bash scripts, hope someone to fix this issue